frontend/packages/dev-console/integration-tests/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "strict": true,
     "esModuleInterop": true,
     "types": ["cypress"],
-    "lib": ["es6", "dom", "es2017"]
+    "lib": ["es6", "dom", "es2017"],
+    "jsx": "react"
```

Comment:
Is this change really required for this PR?

Reply:
Before I have added normal tests not the e2e tests. That time I have added this. Now I removed.